public/apps/ybribe-manifest.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
@Majorfi this is not being applied, didn't look too much into it yet. But if you have any idea then let me know